server/odc-service/src/main/java/com/oceanbase/odc/service/tag/TagService.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.tag;
+
+public interface TagService {
```

Comment:
it is not a general design, may

```java
public interface AccountTags {
  List<Tag> queryTags(String acountName, List<String> labels);
}
```

Reply:
Replace the original `TagService` with a `TagServiceFacade`. Now TagService` only focuses on providing tag service.